nautobot/docs/additional-features/custom-fields.md

```diff
@@ -8,7 +8,9 @@ Within the database, custom fields are stored as JSON data directly alongside ea
 
 ## Creating Custom Fields
 
-Custom fields must be created through the admin UI under Extras > Custom Fields. Nautobot supports these custom field types:
+Custom fields can be created through the admin UI under Extras > Custom Fields or through the REST API.
```

Comment:
In 1.2.0 custom fields have been moved out of the admin UI - we may as well get ahead of this while you're changing it anyways:

```suggestion
Custom fields can be created through the UI under Extensibility > Miscellaneous > Custom Fields or through the REST API.
```

Reply:
Good to know.  Thank you!